src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -30,39 +30,49 @@ class SearchContainer extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.location !== prevProps.location) {
+    const { location } = this.props;
+    if (location !== prevProps.location) {
       global.window.scrollTo(0, 0);
       this.loadSearch();
     }
   }
 
   loadSearch() {
-    const { query } = this.props.location;
-    const searchQuery = query && query.q ? query.q : this.props.searchQuery;
-    const selectedField = query && query.field ? query.field : this.props.searchField;
+    let { searchQuery } = this.props;
```

Comment:
You should add `searchQuery` to the destructuring below and create a new variable for the update instead of overriding the prop value.